PolicyGenerator.go

```diff
@@ -69,12 +69,70 @@ type plugin struct {
 	outputBuffer bytes.Buffer
 }
 
-//nolint: golint
-//noinspection GoUnusedGlobalVariable
-var KustomizePlugin plugin
+func main() {
+	debugFlag := pflag.Bool("debug", false, "Print the stack trace with error messages")
+	pflag.Parse()
+	debug = *debugFlag
+	argpaths := pflag.Args()
+	// When executing with `kustomize build`, need to account for additional argument
+	index := 1
+	// When executing with `./PolicyGenerator` directly
+	if os.Args[0] == "./PolicyGenerator" {
```

Comment:
I think this would be safer. What do you think?
```suggestion
	if path.Base(os.Args[0]) != "kustomize" {
```

Reply:
Yes, definitely.

Reply:
I've been bashing my head against this one until I realized just now that it **is** running `PolicyGenerator` directly under the covers, so this change doesn't work. 🙁

Reply:
So what does `os.Args` look like when it's called from Kustomize?

Reply:
It has the path to the location of the binary, with `PolicyGenerator` at the end. I suppose we could parse that path and assume a user wouldn't be calling the binary in the same way that kustomize is...

Reply:
@dhaiducek perhaps we can just have the caller use the `--debug` flag if they ever want to execute it outside of Kustomize? That'd be safer and clearer I think.

Reply:
I don't think the `debug` flag is the issue here. Kustomize loads the `Args`, with a cached `PolicyGenerator` manifest as the first argument:
```
/Users/dhaiduce/.config/kustomize/plugin/policy.open-cluster-management.io/v1/policygenerator/PolicyGenerator /var/folders/yp/m9m0k3594tz3tpx_ynk8bk240000gn/T/kust-plugin-config-446548996 ./inputFiles ./input/policyGenerator.yaml
```
That temp dir doesn't seem to be easily handled. I think this might be the best route for now...

Reply:
@dhaiducek what I was suggesting is that if the `debug` flag is on, you take `os.Args[1:]`, if not, assume it is run by Kustomize. I just think this check is a little brittle is all.

Reply:
Yeah, either that or an environment variable such as `POLICY_GENERATOR_STANDALONE=true` or something like that.

Reply:
Thanks for the feedback--I've added a `--standalone` flag so that the binary can be run independently.